_component_design/popovers.md

```diff
@@ -5,90 +5,180 @@ secondary: Editing
+
+- For users needing to make changes or take actions while remaining in their original context (for example, a page), a popover allows for small changes or actions to take place without leaving the page.
+- For users needing to be able to view additional details about an item in a workflow, a popover allows easy access to this complex or interactive information without cluttering the page.
 
 {% endcolumn %}
-{% column right:"hxCol-8" %}
-{% figure [caption:"Hover popover"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-hover-specs.svg){:width="100%"}
-{% endfigure %}
-{% figure [caption:"Hover popover specs"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-hover-specs2.svg){:width="100%"}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Meet the popover pattern"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/popovers/popover-hero-image.svg){:width="100%"}
 {% endfigure %}
 {% endcolumn %}
+
 </div>
 
-## Triggered Popover
+## Default layout 
 
 <div class="hxRow">
-{% column left:"hxCol-4" %}
-Overpopovers are triggered as a result of a user hovering over linked area in
-the UI.
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-- font-family: Roboto
-- font-weight: 400
-- font-size: 16px / 1em
-- color: Grey800 (#616161)
+The most defining aspects of a popover are the container and the pointer. The container appears on top of the canvas. The pointer points to the popover trigger. Pointers may be used on either side of the four corners.
+
+- Most popovers have a heading and submit/cancel buttons. If buttons are not needed, a popover should close when the user clicks outside of the popover.
+- The content area may include text, inputs, or interactive content like a table or visualization.
 
 {% endcolumn %}
-{% column left:"hxCol-8" %}
-{% figure [caption:"Triggered popover"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-triggered.svg){:width="100%"}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"A popover consists of a panel, contextual pointer, heading and buttons."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/popovers/popover-default-layout.svg){:width="80%"}
 {% endfigure %}
-{% figure [caption:"Triggered popover specs"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-triggered-specs.svg){:width="100%"}
+{% endcolumn %}
+</div>
+
+## Pattern specifications 
+
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Popover Spacing
+
+
+For specifications on laying out form fields and body text within a popover, refer to [Inputs and Controls]({{site.baseurl}}/components/inputs-and-controls.html).
```

Comment:
See [Inputs and Controls] for specifications that describe laying out form fields and body text within a popover.

Reply:
updated.